release notes/v0.30.0.md

```diff
@@ -0,0 +1,123 @@
+k6 v0.30.0 is here! :tada: It was a bit of a slow after-holiday release, but it still packs a few major new features and improvements that users have been requesting for a long time!
+
+## New features
+
+### Shared memory read-only arrays between VUs ([#1739](https://github.com/loadimpact/k6/pull/1739))
+
+k6 has long had an issue with the handling of big data files with test fixtures. For example, if you have a huge `users.json` file with test users for your application:
+```js
+[
+  {"username": "user1", "password": "password1", "other": "some-long-data-....-1"},
+  {"username": "user2", "password": "password2", "other": "some-long-data-....-2"},
+  // ... ~1 million more users or more... :D
+  {"username": "user999999", "password": "password999999", "other": "some-long-data-....-999999"}
+]
+```
+
+If you just use `JSON.parse(open('users.json'))` in your script, then every VU will have a copy of the whole huge data set. Every VU in k6 is a separate JavaScript runtime, so there wasn't a thread-safe way to share data between them. Until now, that is!
```

Comment:
Technically there wasn't "a way" to share data ... even non-thread-safe-ly;P . So maybe just drop the `thread-safe` part?

Reply:
There was, with the setup data, until we saw the data races and fixed it :smile: I'd prefer to leave thread-safe, to emphasize that a special solution was actually needed